sci-visualization/xgraph/xgraph-12.1-r4.ebuild

```diff
@@ -1,21 +1,22 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
```

Comment:
The EAPI bump requires a revision bump of the ebuild.

Reply:
Sure, i've updated the PR and moved the ebuild to `r5`
Since it doesn't have any stable keywords i didn't keep the `r4`. Hope thats fine.